src/IIIFPresentation/API/Infrastructure/ControllerBaseX.cs

```diff
@@ -170,10 +171,11 @@ public static ObjectResult Forbidden(this ControllerBase controller)
         => controller.PresentationProblem(statusCode: (int)HttpStatusCode.Forbidden);
     
     /// <summary>
-    /// Create a 201 result with standard serialized value or custom serialized IIIF object if value is
+    /// Creates either a 201 or 202 result with standard serialized value or custom serialized IIIF object if value is
```

Comment:
Tiny thing but there's nothing limiting this to 201 or 202, it creates with whatever status code is specified.